fe/fe-core/src/test/java/com/starrocks/analysis/AlterRoutineLoadStmtTest.java

```diff
@@ -107,7 +107,8 @@ public void testParser() {
                 + "(\n"
                 + "\"kafka_partitions\" = \"0, 1, 2\",\n"
                 + "\"kafka_offsets\" = \"100, 200, 100\",\n"
-                + "\"property.group.id\" = \"group1\"\n"
+                + "\"property.group.id\" = \"group1\",\n"
+                + "\"confluent.schema.registry.url\" = \"https://key:passwrod@addr\"\n"
```

Comment:
What's alternation supported? add/drop or modify?

Reply:
> What's alternation supported? add/drop or modify?

modify

Reply:
You should add it into the commit message.